File: 100-Days-Series/Day 31/ToggleCase.java
class ToggleCase{
    // Method for ToggleCase
    public static String getToggleCase(String str){
        StringBuilder result=new StringBuilder();
        for(char ch:str.toCharArray()){
            if(Character.isLowerCase(ch)){
                result.append(Character.toUpperCase(ch));
            }else if(Character.isUpperCase(ch)){
                result.append(Character.toLowerCase(ch));
            }else{
                result.append(ch);
            }
        }
        return result.toString();
    }
    // Main Method
    public static void main(String [] args){
        // Let's Start here
        String str;
        java.util.Scanner sc=new java.util.Scanner(System.in);
        System.out.print("Enter a string : ");
        str=sc.nextLine();
        String toggledString=getToggleCase(str);
        System.out.println("Original String : "+str);
        System.out.println("Toggled String : "+toggledString);
    }
}

File: 100-Days-Series/Day 31/ToggleCase.js
function isLowerCase(char) {
    return char === char.toLowerCase() && char !== char.toUpperCase();
}
function isUpperCase(char) {
    return char === char.toUpperCase() && char !== char.toLowerCase();
}
//Arrow Function for ToggleCase
const getToggleCase=(str)=>{
    result="";
    for(let i=0;i<str.length;i++){
        let ch=str.charAt(i);
        if(isUpperCase(ch)){
            result+=ch.toLowerCase();
        }else if(isLowerCase(ch)){
            result+=ch.toUpperCase();
        }else{
            result+=ch;
        }
    }
    return result;
}
// Let's Start here
let str="Hello, World!";
console.log("Original String : "+str);
toggledString=getToggleCase(str);
console.log("Toggled String : "+toggledString);